src/components/Contributors.tsx

```diff
@@ -62,6 +67,7 @@ const Contributors = () => {
         <DialogContent onOpenAutoFocus={(e) => e.preventDefault()}>
           <Card className="border-none -m-5">
             <CardHeader>
+              <p>Chords:{Version}</p>
```

Comment:
I recommend to put software name in env as well

Reply:
I have done changes, please review.